### ISSUE TITLE
New theme dialog should autosize

### ISSUE THREAD

**Issue by peller** [maqetta/maqetta#1345]
New Theme pops up a dialog

On the Mac, you can zoom in and out with pinch gestures on the trackpad which change the font size.  Do this enough and the contents of the dialog will not fit.  This is probably because a fixed size was assigned to the dialog.  It should be able to auto-size based on its contents.

**Comment by JonFerraiolo** [comment]
@childsb, @billreed63, @jhpedemonte 

All dialogs need to auto-size (ie no explicit width/height) both for accessibility and internationalization reasons (e.g., German labels tend to be longer, Japanese labels tend to be shorter). In the past couple of months, I changed a few dialogs from fixed size to auto size in order to address various UI problems. It's usually fairly easy to change an existing dialog to auto-size. Sometimes it's necessary to add 100% to a couple of widgets or HTML elements, or sometimes you need to add float:right.

**Comment by aerwin** [comment]
--> no milestone to M7
--> @doronrosenberg 
--> prio low

Perhaps no longer an issue with resizable dialogs?
